.internal/aws/cloudformation/application.yaml

```diff
@@ -0,0 +1,85 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  Elastic Serverless Forwarder
+
+  SAM Template for the application, not intended to be deployed on its own
```

Comment:
@ravikesarwani
please, let me know if you want to improve the wording of this sentence in order to instruct the users that this specific application is not the one to be deployed